writer.go

```diff
@@ -422,6 +422,34 @@ func (p *MediaPlaylist) Encode() *bytes.Buffer {
 		}
 	}
 
+	// default key (workaround for Widevine)
+	if false && p.Key != nil {
```

Comment:
I don't fully understand why you need to add this block if it is disabled anyway. :thinking:

Reply:
Look at the full PR, this is the only place where I disable it.